db/migrate/20170103144759_encrypt_user_credentials.rb

```diff
@@ -0,0 +1,6 @@
+class EncryptUserCredentials < ActiveRecord::Migration[5.0]
+  def change
+      rename_column :user_credentials, :credential_value, :encrypted_credential_value
```

Comment:
Actually, no need to rename the column, I missed https://github.com/attr-encrypted/attr_encrypted#the-attribute-option.

I should check what happens with `attr_encrypted` if I pass it an empty key. Combined with https://github.com/attr-encrypted/attr_encrypted#the-if-and-unless-options this should make the encryption / decryption process transparent whether there is an already set `ENV['APP_ENCRYPTION_PASSPHRASE']` or not.